libc/src/__support/math_extras.h

```diff
@@ -14,7 +14,6 @@
 #include "src/__support/CPP/limits.h"        // CHAR_BIT, numeric_limits
 #include "src/__support/CPP/type_traits.h"   // is_unsigned_v
 #include "src/__support/macros/attributes.h" // LIBC_INLINE
-#include "src/__support/macros/config.h"     // LIBC_HAS_BUILTIN
```

Comment:
This change lets us remove the dependency on `libc.src.__support.macros.config` from `math_extras` in libc/src/__support/CMakeLists.txt, and `:__support_macros_config` from `__support_math_extras` from utils/bazel/llvm-project-overlay/libc/BUILD.bazel.

Reply:
Done.